### PR TITLE
Proposed Migration Notes

### PR DESCRIPTION
**Pull request by davetcoleman**
Please leave feedback on the described process - what did I forget to document?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+All of the git history and the current three supported branches (indigo|jade|kinetic) have been migrated to the single repo [https://github.com/ros-planning/moveit](https://github.com/ros-planning/moveit) for the following repos:
+
+ - https://github.com/ros-planning/moveit_core
+ - https://github.com/ros-planning/moveit_ros
+ - https://github.com/ros-planning/moveit_planners
+ - https://github.com/ros-planning/moveit_plugins
+ - https://github.com/ros-planning/moveit_setup_assistant
+ - https://github.com/ros-planning/moveit_commander
+ - https://github.com/ros-planning/moveit_ikfast
+ - https://github.com/ros-planning/moveit_resources
```
Comment (jrgnicho) [inline README.md:12]: Wasn't moveit_experimental part of this list too?

Comment (davetcoleman) [inline README.md:12]: thanks, fixed

File: README.md [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+
+As of **Friday August 5th** (subject to be pushed back if necessary) these repos will be fully deprecated and all code changes / PRs should be created or moved to the single *moveit* repo.
+
+The rational for this migration is further discussed on [Discourse](http://discourse.ros.org/t/migration-to-one-github-repo-for-moveit/266).
+
+## Merge Method
+
+We used a common ``git merge`` method that was loosely inspired by [this blog](https://saintgimp.org/2013/01/22/merging-two-git-repositories-into-one-repository-without-losing-file-history/) to preserve the git history of all repos. The automated merge script we used can be found [here](https://github.com/davetcoleman/moveit_merge/blob/master/git_merge_moveit.sh).
+
+Unfortuantly on Github we will not be able to see the git history for files - this is a limitation of Github and has been discussed online. In fact it is also a limitation of git - ``git`` has no actual file moving capabailites but intead detects changes automatically similar to refactoring a single file. Locally you can still see the history, however, using the ``--follow`` command e.g.:
```
Comment (jrgnicho) [inline README.md]: Typo _Unfortuantly_

Comment (davetcoleman) [inline README.md]: fixed

**Comment by jrgnicho** [comment]
+1 from me

File: README.md [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+
+In the future we are interested in flattening out some of the subfolders - i.e. "moveit/moveit_ros/planning" could become "moveit/moveit_ros_planning" since this better represents the package name. But this is not part of the initial migration.
+
+## Moving Issues Over
+
+We would like to start opening all new issues on the new repo, and old ones should be copied there as well. This will be one of the objectives of World MoveIt! Day.
+
+## Releasing Debians
+
+Future releasing of indgo|jade|kinetic we be from this new unified repo.
```
Comment (dornhege) [inline README.md]: we -> will

**Comment by dornhege** [comment]
+1

File: README.md [hunk 4]
```diff
@@ -1,3 +1,61 @@
...
+
+As of **Friday August 5th** (subject to be pushed back if necessary) these repos will be fully deprecated and all code changes / PRs should be created or moved to the single *moveit* repo.
+
+The rational for this migration is further discussed on [Discourse](http://discourse.ros.org/t/migration-to-one-github-repo-for-moveit/266).
+
+## Merge Method
+
+We used a common ``git merge`` method that was loosely inspired by [this blog](https://saintgimp.org/2013/01/22/merging-two-git-repositories-into-one-repository-without-losing-file-history/) to preserve the git history of all repos. The automated merge script we used can be found [here](https://github.com/davetcoleman/moveit_merge/blob/master/git_merge_moveit.sh).
+
+Unfortunately on Github we will not be able to see the git history for files - this is a limitation of Github and has been discussed online. In fact it is also a limitation of git - ``git`` has no actual file moving capabailites but intead detects changes automatically similar to refactoring a single file. Locally you can still see the history, however, using the ``--follow`` command e.g.:
```
Comment (rhaschke) [inline README.md]: intead -> instead

Comment (davetcoleman) [inline README.md]: thanks!

File: README.md (hunk 3 above)
Comment (davetcoleman) [inline README.md]: thanks!

**Pull request merged**